docs/source/reference/modding/settings/physics.rst

```diff
@@ -0,0 +1,13 @@
+Physics Settings
+################
+
+physics framerate
+---------------
+
+:Type:		floating point
+:Range:		10.0 to 60.0
+:Default:	60.0
+
+Allow to set how frequently an engine will do physics calculations. A default falue is 60 times per second.
+This setting allow developers with low-clocked CPUs to test an engine.
+Changing from default value can lead to physics bugs. Change this setting on your own risk, and reset a value to default before filling a physics-related bugreport!
```

Comment:
Such as?  You tested for awhile at 30, did you notice any?

Reply:
I did not notice any problem, but maybe I just missed something. Take a look at [this](https://forum.openmw.org/viewtopic.php?f=8&t=4318) topic.